### PR TITLE
Consistent modal paddings

### PR DESCRIPTION
**Pull request by joel-jeremy**
Use the consistent paddings for modals

### CODE REVIEW

File: packages/desktop-client/src/components/modals/ReportBudgetSummaryModal.tsx [hunk 1]
```diff
@@ -23,26 +23,32 @@ export function ReportBudgetSummaryModal({
 }: ReportBudgetSummaryModalProps) {
   const currentMonth = monthUtils.currentMonth();
   return (
-    <Modal title="Budget Summary" {...modalProps}>
-      {() => (
```
Comment (joel-jeremy) [inline packages/desktop-client/src/components/modals/ReportBudgetSummaryModal.tsx:27]: Removed lambda

File: packages/desktop-client/src/components/modals/RolloverBudgetSummaryModal.tsx [hunk 2]
```diff
@@ -78,28 +78,34 @@ export function RolloverBudgetSummaryModal({
   };
 
   return (
-    <Modal title="Budget Summary" {...modalProps}>
-      {() => (
```
Comment (joel-jeremy) [inline packages/desktop-client/src/components/modals/RolloverBudgetSummaryModal.tsx:82]: Removed lambda

File: packages/desktop-client/src/components/modals/SingleInputModal.tsx [hunk 3]
```diff
@@ -45,58 +45,52 @@ export function SingleInputModal({
...
-      padding={0}
+      padding={10}
       style={{
         flex: 1,
-        padding: '0 10px',
-        paddingBottom: 10,
         borderRadius: '6px',
       }}
     >
-      {() => (
```
Comment (joel-jeremy) [inline packages/desktop-client/src/components/modals/SingleInputModal.tsx:56]: Removed lambda

File: packages/desktop-client/src/components/modals/SwitchBudgetTypeModal.tsx [hunk 4]
```diff
@@ -28,45 +28,51 @@ export function SwitchBudgetTypeModal({
       }
     : {};
   return (
-    <Modal title="Switch budget type?" {...modalProps}>
-      {() => (
```
Comment (joel-jeremy) [inline packages/desktop-client/src/components/modals/SwitchBudgetTypeModal.tsx:32]: Removed lambda

File: packages/desktop-client/src/components/modals/TransferModal.tsx [hunk 5]
```diff
@@ -71,63 +71,57 @@ export function TransferModal({
...
-      padding={0}
+      padding={10}
       style={{
         flex: 1,
-        padding: '0 10px',
-        paddingBottom: 10,
         borderRadius: '6px',
       }}
     >
-      {() => (
```
Comment (joel-jeremy) [inline packages/desktop-client/src/components/modals/TransferModal.tsx:82]: Removed lambda

File: packages/desktop-client/src/components/modals/Notes.tsx [hunk 6]
```diff
@@ -41,57 +41,53 @@ export function Notes({ modalProps, id, name, onSave }: NotesProps) {
...
-      padding={0}
+      padding={10}
       style={{
         flex: 1,
         height: '50vh',
-        padding: '0 10px',
         borderRadius: '6px',
       }}
     >
-      {() => (
```
Comment (joel-jeremy) [inline packages/desktop-client/src/components/modals/Notes.tsx:52]: Removed lambda

File: packages/desktop-client/src/components/modals/HoldBufferModal.tsx [hunk 7]
```diff
@@ -42,48 +42,42 @@ export function HoldBufferModal({
...
-      padding={0}
+      padding={10}
       style={{
         flex: 1,
-        padding: '0 10px',
-        paddingBottom: 10,
         borderRadius: '6px',
       }}
     >
-      {() => (
```
Comment (joel-jeremy) [inline packages/desktop-client/src/components/modals/HoldBufferModal.tsx:53]: Removed lambda

File: packages/desktop-client/src/components/modals/CoverModal.tsx [hunk 8]
```diff
@@ -74,45 +74,39 @@ export function CoverModal({
...
-      padding={0}
+      padding={10}
       style={{
         flex: 1,
-        padding: '0 10px',
-        paddingBottom: 10,
         borderRadius: '6px',
       }}
     >
-      {() => (
```
Comment (joel-jeremy) [inline packages/desktop-client/src/components/modals/CoverModal.tsx:85]: Removed lambda

File: packages/desktop-client/src/components/modals/AccountAutocompleteModal.tsx [hunk 9]
```diff
@@ -42,11 +42,10 @@ export function AccountAutocompleteModal({
       focusAfterClose={false}
       {...modalProps}
       onClose={_onClose}
-      padding={0}
+      padding={10}
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/modals/AccountAutocompleteModal.tsx:45]: **❓ question:** would it make sense to move the `padding=10` inside the `Modal` component as the default value? Seems like it's used universally almost everywhere now.

Comment (MatissJanis) [reply packages/desktop-client/src/components/modals/AccountAutocompleteModal.tsx:45]: Also `style = { flex: 1, borderRadius: '6px' }`

**Review by MatissJanis — DISMISSED** [review]

**Review by MatissJanis — DISMISSED** [review]

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**